scripts/bump-version-file.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('node:fs/promises');
+
+exports.preCommit = async (props) => {
+  const updatedVersion = (await fs.readFile('./src/VERSION.ts', {
+    encoding: 'utf-8'
+  })).replace(/'[^']*'/g, `'${props.version}'`)
```

Comment:
`props.version` will be the new version to be published so this will always match the one in `package.json`
https://github.com/TriPSs/conventional-changelog-action/?tab=readme-ov-file#pre-commit-hook